applications/matter_bridge/README.rst

```diff
@@ -0,0 +1,327 @@
+.. _matter_bridge_app:
+
+Matter: Bridge
+################
```

Comment:
```suggestion
Matter bridge
##############
```
I would treat this on-par with "Matter controller" and other network topology devices.
See https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/protocols/matter/overview/network_topologies.html#matter-network-concepts.
Also because, assuming that "Matter:" is here to indicate the protocol and the second part is the device type, this could be interpreted we're adding an application for controlling bridges. ;)